### PR TITLE
feat: Implemented most necessary features

### PR DESCRIPTION
**Pull request by Arjun-G-04**
- Implemented parallax effect for footer images
- Fixed bug: image overflow in hero and footer section when scaling

### CODE REVIEW

**Comment by Arjun-G-04** [comment]
Uhh.. found a bug, working on it. Not ready for merge yet.

**Comment by Arjun-G-04** [comment]
Fixed the bug, its fine now to merge hopefully 😅

**Comment by Arjun-G-04** [comment]
feat: Implemented most necessary features

- Made all pages responsive (including Navbar)
- Added animation to about text
- Fine tuned footer parallax
- Implemented timeline animation (progress bar + deer)
- Fixed some alignment probs in testimonial cards

File: package.json
Comment (Sesha-2804) [inline package.json:1]: - Remove unused packages like **framer-motion** etc from the repository.

File: src/components/About/About.tsx
Comment (Sesha-2804) [inline src/components/About/About.tsx:1]: - Create a `content.ts` file and put all the content into it, then import it whenever and wherever required.
- Fill `alt` in all the `img` tag, don't leave it empty.

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -9,27 +9,30 @@ import { useRef } from "react";
...
-			<div className="w-full basis-[20%] items-center justify-end p-8 flex flex-col z-50 shrink-0 bg-[#234356]">
-				<span className=" font-quicksand text-white font-light text-4xl">
-					Made with love by Delta Force
+			<div className="w-full basis-[10%] items-center justify-end p-8 flex flex-col z-50 shrink-0 bg-[#234356]">
+				<span className="font-quicksand text-white font-light text-2xl sm:text-4xl text-center">
+					Made with <span className="text-red">&hearts;</span> by{" "}
+					<a href="https://delta.nitt.edu" target="_blank">
+						<span className="text-green">Delta Force</span>
+					</a>
 				</span>
```
Comment (Sesha-2804) [inline src/components/Footer/Footer.tsx]: - [x] Change the heart icon, you may copy paste from other repositories.
- [x] Change the Delta Force `text colour` to Delta green (Again you may take it from the official website)
- [x] Add `animate-pulse` to the heart.

File: src/components/Hero/Hero.tsx
Comment (Sesha-2804) [inline src/components/Hero/Hero.tsx:1]: Screenshot from 2023-09-10 21-05-57

- [x] _What is DWoC_ section overlaps and cuts the `Hero` section. Try changing it.
- [x] Fill `alt` in all the `img` tag, don't leave it empty.

File: src/components/NavMenu/NavMenu.tsx [hunk 2]
```diff
@@ -43,64 +46,99 @@ const externalLink: NavItem[] = [
...
+						className={`w-[2px] transition-all transform ease-in-out duration-500 ${
+							isOpen
+								? "-rotate-[135deg] h-[20%] -translate-x-[1px] group-hover:-translate-x-[2px] group-hover:h-[15%] group-hover:duration-200 group-hover:w-[4px] bg-menu-dark"
+								: " bg-accent h-[40%] animate-bounce-1 "
+						}`}
+					></span>
+				</div>
+			</div>
+			<div className="fixed sm:hidden top-0 h-14 w-full z-[90] flex justify-center items-end">
+				<div className="flex flex-row justify-between px-5 items-center rounded-lg shadow-[0_10px_60px_10px_rgba(0,0,0,0.6)] bg-menu-dark/60 backdrop-blur-sm w-[95%] h-[90%]">
```
Comment (Sesha-2804) [inline src/components/NavMenu/NavMenu.tsx]: - [x] Change `bg-menu-dark/60 ` to `bg-menu-dark`.

File: src/components/Testimonials/Testimonials.tsx
Comment (Sesha-2804) [inline src/components/Testimonials/Testimonials.tsx:1]: - [x] Move all the `testimonialsData` to `content.ts` file and import from there.

File: src/components/Timeline/Timeline.tsx
Comment (Sesha-2804) [inline src/components/Timeline/Timeline.tsx:1]: - [x] Add some vertical space between **T** of _Timeline_ and the line. Horizontal alignment looks fine just add some vertical space.(Mobile)
- [x] Width of the line could be reduced.(Mobile)

File: tailwind.config.js
Comment (Sesha-2804) [inline tailwind.config.js:1]: - [x] Remove unused lines if there are any.

**Review by Sesha-2804 — COMMENTED** [review]
@Arjun-G-04 @AkashBalcham Try to do all the requsted changes ASAP and ping me once you are done.

**Comment by Arjun-G-04** [comment]
@Sesha-2804 completed the changes bro

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -9,27 +9,31 @@ import { useRef } from "react";
...
-				<span className=" font-quicksand text-white font-light text-4xl">
-					Made with love by Delta Force
+			<div className="w-full basis-[10%] items-center justify-end p-8 flex flex-col z-50 shrink-0 bg-[#234356]">
+				<span className="font-quicksand text-white font-light text-2xl sm:text-4xl text-center">
+					Made with{" "}
+					<span className="text-red animate-pulse">&#x2764;</span> by{" "}
+					<a href="https://delta.nitt.edu" target="_blank">
+						<span className="text-green">Delta Force</span>
+					</a>
 				</span>
```
Comment (Sesha-2804) [inline src/components/Footer/Footer.tsx]:
```
<span className="text-white font-light text-2xl sm:text-4xl text-center select-none">
					<span className="font-quicksand">Made with </span>
					<span className="text-red animate-pulse">&#x2764;</span>
					<span className="font-quicksand"> by </span>
					<a href="https://delta.nitt.edu" target="_blank">
						<span className="text-green font-quicksand font-medium">
							Delta Force
						</span>
					</a>
				</span>
``` 
Replace with this

File: src/pages/Landing/Landing.tsx [hunk 4]
```diff
@@ -6,12 +6,29 @@ import {
...
+			setIsClicked(!isClicked);
+		}
+	};
+
 	return (
-		<div className="w-screen h-screen bg-white relative">
-			<NavMenu />
+		<div
+			onClick={handleClick}
+			className="w-screen h-screen bg-white relative"
```
Comment (Sesha-2804) [inline src/pages/Landing/Landing.tsx]: Add `select-none` here

**Pull request merged**